### PR TITLE
added license and readme

### PR DESCRIPTION
**Pull request by kxwu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+### LaunchDarkly Sample Python Application  ###
+We've built a simple console application that demonstrates how LaunchDarkly's SDK works.  Below, you'll find the basic build procedure, but for more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/).
+##### Build instructions  #####
+1. Install the LaunchDarkly Python SDK by running `pip install ldclient-py==0.19.1`
```
Comment (jkodumal) [inline README.md]: @apucacao This repo has a `requirements.txt` file. I think the quickstart should instruct people to create and save `requirements.txt`, and then `pip install -r requirements.txt`. Much like what the ruby SDK does.

@kxwu if AG agrees, then this should just be `pip install -r requirements.txt`

**Comment by jkodumal** [comment]
One comment then :+1:

File: README.md (hunk 1 above)
Comment (apucacao) [inline README.md]: Oh right, I'll update.

**Comment by apucacao** [comment]
lgtm

**Pull request merged**